Sources/Purchasing/ReceiptFetcher.swift

```diff
@@ -71,7 +71,7 @@ private extension ReceiptFetcher {
         let firstOSVersionWithoutBug: OperatingSystemVersion = OperatingSystemVersion(majorVersion: 7,
                                                                                       minorVersion: 0,
                                                                                       patchVersion: 0)
-        let isBelowFirstOSVersionWithoutBug = ProcessInfo.processInfo.isOperatingSystemAtLeast(firstOSVersionWithoutBug)
+        let isBelowFirstOSVersionWithoutBug = !ProcessInfo.processInfo.isOperatingSystemAtLeast(firstOSVersionWithoutBug)
```

Comment:
Great catch!! This was introduced during the Swift migration, it [is correct in v3](https://github.com/RevenueCat/purchases-ios/blob/3.14.2/Purchases/Purchasing/RCReceiptFetcher.m#L27). 

Thanks so much for reporting and fixing!! 🎉 👏 🏆 

Merging and we'll get it shipped soon!